### ISSUE TITLE
gantt chart

### ISSUE THREAD

**Issue by juuuuuuuuuuuuuu** [recharts/recharts#813]
i want to make like gantt chart
how to make staring point difference?
i will use  the stackbarchart ...
please help me...

**Comment by JacquiManzi** [comment]
I tried for a bit to make a gantt chart with Recharts using a horizontal stacked bar chart. From what I recall, recharts was not flexible enough in terms of data requirements for stacked bar charts.
Check out this issue: https://github.com/recharts/recharts/issues/753

Also, using the x-axis for time on a horizontal stacked bart chart in Recharts was not worth the effort.

In the end, I made a gantt chart using pure D3 and it was extremely easy.
